src/lib/menu/menu-panel.ts

```diff
@@ -37,6 +37,16 @@ export interface MatMenuPanel<T = any> {
   lazyContent?: MatMenuContent;
   backdropClass?: string;
   hasBackdrop?: boolean;
+
+  /**
+   * @deprecated To be removed.
+   * @deletion-target 8.0.0
```

Comment:
```suggestion
   * @breaking-change 8.0.0
```

There are two lint failures for `removeItem` and `addItem`

Reply:
Thanks, fixed.